var/spack/repos/builtin/packages/rocm-smi-lib/package.py

```diff
@@ -13,17 +13,19 @@ class RocmSmiLib(CMakePackage):
        for applications to monitor and control GPU applications."""
 
     homepage = "https://github.com/RadeonOpenCompute/rocm_smi_lib"
-    url      = "https://github.com/RadeonOpenCompute/rocm_smi_lib/archive/rocm-3.5.0.tar.gz"
+    url      = "https://github.com/RadeonOpenCompute/rocm_smi_lib/archive/rocm-3.7.0.tar.gz"
 
     maintainers = ['srekolam', 'arjun-raj-kuppala']
 
+    version('3.7.0', sha256='72d2a3deda0b55a2d92833cd648f50c7cb64f8341b254a0badac0152b26f1391')
     version('3.5.0', sha256='a5d2ec3570d018b60524f0e589c4917f03d26578443f94bde27a170c7bb21e6e')
 
     variant('build_type', default='Release', values=("Release", "Debug"), description='CMake build type')
+
     depends_on('cmake@3:', type='build')
 
     @run_after('install')
     def post_install(self):
         shutil.rmtree(self.prefix.lib)
-        install_tree(self.prefix.rocm_smi,  self.prefix)
+        copy_tree(self.prefix.rocm_smi,  self.prefix)
```

Comment:
Why the change here? The only difference between `copy_tree` and `install_tree` is how permissions are handled.

Reply:
there was sync issue between my branch. reverted back to install_tree.